### ISSUE TITLE
<input>s and <select>s should have the same focus style

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/vscode#2349]
Testing #2145

* a `<select>` gets a blue focus border:

![image](https://cloud.githubusercontent.com/assets/5047891/12582469/a1f0d004-c43d-11e5-919e-8f3a38a45b59.png)

* while a `<input>` does not

We should see what is best and be consistent, either both get a focus border or not.
